src/index.js

```diff
@@ -33,13 +33,18 @@ each(events, event => {
 })
 
 export default class Timeline extends Component {
-
-  componentWillMount() {
+  constructor(props) {
+    super(props);
     this.state = {
-      // NOTE we store custom times on the state to enable us to diff with new
-      // custom times and add or remove the elements with visjs
       customTimes: []
-    }
+    };
+  }
+  componentWillMount() {
+    // NOTE we store custom times on the state to enable us to diff with new
+    // custom times and add or remove the elements with visjs
+    this.setState({
+      customTimes: []
+    });
```

Comment:
I'm not a contributor, but just wanted to ask why this `setState` call is still needed in `componentWillMount` when the state is already being set in the constructor? Can't this lifecycle method be dropped completely now?